Cython/Compiler/Options.py

```diff
@@ -787,7 +787,7 @@ def to_fingerprint(item):
     evaluate_tree_assertions=False,
     emit_linenums=False,
     relative_path_in_code_position_comments=True,
-    c_line_in_traceback=True,
+    c_line_in_traceback=None,
```

Comment:
I think we now also have to declare the type of the option as `bool`.

Reply:
Not sure I understand this comment? Possibly you're referring to `directive_types` (which applies to directives rather than options?)

Reply:
Yes, that's what I meant. Sorry, I mixed up the two.

Then we still need to make sure that we receive a boolean value, regardless of what users pass us. We shouldn't report misuse by a failing `int()` call, and we shouldn't let users pass arbitrary integer values into the C code.